tests/test_sycl_queue.py

```diff
@@ -1283,111 +1283,108 @@ def test_out_multi_dot(device):
         assert_sycl_queue_equal(result.sycl_queue, exec_q)
 
 
-@pytest.mark.parametrize(
-    "func", ["fft", "ifft", "rfft", "irfft", "hfft", "ihfft"]
-)
-@pytest.mark.parametrize(
-    "device",
-    valid_devices,
-    ids=[device.filter_string for device in valid_devices],
-)
-def test_fft(func, device):
-    dtype = numpy.float64 if func in ["rfft", "ihfft"] else numpy.complex128
-    data = numpy.arange(20, dtype=dtype)
-    dpnp_data = dpnp.array(data, device=device)
-
-    expected = getattr(numpy.fft, func)(data)
-    result = getattr(dpnp.fft, func)(dpnp_data)
-    assert_dtype_allclose(result, expected, factor=16)
-
-    expected_queue = dpnp_data.get_array().sycl_queue
-    result_queue = result.get_array().sycl_queue
-    assert_sycl_queue_equal(result_queue, expected_queue)
-
-
-@pytest.mark.parametrize(
-    "device",
-    valid_devices,
-    ids=[device.filter_string for device in valid_devices],
-)
-def test_fftn(device):
-    data = numpy.arange(24, dtype=numpy.complex64).reshape(2, 3, 4)
-    dpnp_data = dpnp.array(data, device=device)
+class TestFft:
+    @pytest.mark.parametrize(
+        "func", ["fft", "ifft", "rfft", "irfft", "hfft", "ihfft"]
+    )
+    @pytest.mark.parametrize(
+        "device",
+        valid_devices,
+        ids=[device.filter_string for device in valid_devices],
+    )
+    def test_fft(self, func, device):
+        dtype = numpy.float64 if func in ["rfft", "ihfft"] else numpy.complex128
+        data = numpy.arange(20, dtype=dtype)
+        dpnp_data = dpnp.array(data, device=device)
 
-    expected = numpy.fft.fftn(data)
-    result = dpnp.fft.fftn(dpnp_data)
-    assert_dtype_allclose(result, expected, check_only_type_kind=True)
+        expected = getattr(numpy.fft, func)(data)
+        result = getattr(dpnp.fft, func)(dpnp_data)
+        assert_dtype_allclose(result, expected, factor=24)
 
-    expected_queue = dpnp_data.sycl_queue
-    result_queue = result.sycl_queue
-    assert_sycl_queue_equal(result_queue, expected_queue)
+        expected_queue = dpnp_data.get_array().sycl_queue
+        result_queue = result.get_array().sycl_queue
```

Comment:
dpnp.ndarray also has property `sycl_queue`:
```suggestion
        expected_queue = dpnp_data..sycl_queue
        result_queue = result..sycl_queue
```